From 36996625faa2e352c846183e74ab3797fcd10e3a Mon Sep 17 00:00:00 2001
From: Joseph Myers <joseph@codesourcery.com>
Date: Fri, 1 Sep 2017 17:14:43 +0000
Subject: [PATCH] Include <string.h> for strcmp call from
 sysdeps/unix/sysv/linux/gentempfd.c.

	* sysdeps/unix/sysv/linux/gentempfd.c: Include <string.h>.
---
 ChangeLog                           | 4 ++++
 sysdeps/unix/sysv/linux/gentempfd.c | 1 +
 2 files changed, 5 insertions(+)

diff --git a/ChangeLog b/ChangeLog
index 4f8a12c..2d681d1 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,7 @@
+2017-09-01  Joseph Myers  <joseph@codesourcery.com>
+
+	* sysdeps/unix/sysv/linux/gentempfd.c: Include <string.h>.
+
 2017-09-01  Adhemerval Zanella  <adhemerval.zanella@linaro.org>
 	    Andreas Schwab  <schwab@suse.de>
 
diff --git a/sysdeps/unix/sysv/linux/gentempfd.c b/sysdeps/unix/sysv/linux/gentempfd.c
index 902cbe2..ff3a649 100644
--- a/sysdeps/unix/sysv/linux/gentempfd.c
+++ b/sysdeps/unix/sysv/linux/gentempfd.c
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <fcntl.h>
 #include <errno.h>
+#include <string.h>
 
 int
 __gen_tempfd (int flags)
-- 
2.7.4

